tf2onnx/tfonnx.py

```diff
@@ -466,7 +460,7 @@ def compat_handler(ctx, node, **kwargs):
                  rewrite_single_direction_lstm, rewrite_bi_direction_lstm,
                  rewrite_single_direction_gru, rewrite_bi_direction_gru,
                  rewrite_custom_rnn_cell, rewrite_generic_loop, rewrite_cond,
-                 rewrite_biasadd_with_conv2d,
+                 rewrite_biasadd_with_conv2d, rewrite_constant_fold
```

Comment:
might help to run const_cold first in the longer term but maybe more risk right now

Reply:
Moved it to first rewriter.